packages/teleport-project-generator-gatsby/src/utils.ts

```diff
@@ -69,6 +69,11 @@ const generateHTMLNode = (uidl: ProjectUIDL, options: EntryFileOptions, t = type
   const headNode = ASTBuilders.createJSXTag('head')
   const bodyNode = ASTBuilders.createJSXTag('body')
   const noScriptNode = ASTBuilders.createJSXTag('noscript')
+  const manifestTag = ASTBuilders.createJSXTag('link')
```

Comment:
I think you should check here if uidl.globals.manifest exists, otherwise it doesn't make sense to add the manifest link

Reply:
yes, should add that 👍